src/helper.ts

```diff
@@ -9,10 +10,35 @@ export const findVariant = (type: string) =>
 export const capitalizeFirstLetter = (string: string) =>
   string.charAt(0).toUpperCase() + string.slice(1);
 
-export const getFromLocale = (name: string): Array<number> =>
-  JSON.parse(localStorage.getItem(name)!);
+export const getFromLocale = (name: string) => localStorage.getItem(name);
 
 export const saveInLocal = <T>(name: string, date: T) =>
   localStorage.setItem(name, JSON.stringify(date));
 
 export const clearLocal = (name: string) => localStorage.removeItem(name);
+
+export const showToastError = (text: string) => {
+  toast.error(text, {
+    autoClose: 3000,
+    position: 'bottom-right',
+    theme: 'colored',
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+    progress: undefined,
+  });
+};
+
+export const showToastSuccess = (text: string) => {
+  toast.success(text, {
+    autoClose: 3000,
+    position: 'bottom-right',
+    theme: 'colored',
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+    progress: undefined,
+  });
```

Comment:
я бы предложила добавить второй аргумент. Почему - чтобы дать возможно из вне менять конфиг, если нужно
```suggestion
const toastConfig = {
    autoClose: 3000,
    position: 'bottom-right',
    theme: 'colored',
    hideProgressBar: false,
    closeOnClick: true,
    pauseOnHover: true,
    draggable: true,
    progress: undefined,
}
export const showToastSuccess = (text: string, ...config = {} ) => {
  toast.success(text, {
    ...toastConfig,
    ...config
  });
```